src/gtest/test_foundersreward.cpp

```diff
@@ -0,0 +1,173 @@
+#include <gtest/gtest.h>
+
+#include "main.h"
+#include "utilmoneystr.h"
+#include "chainparams.h"
+#include "utilstrencodings.h"
+#include "zcash/Address.hpp"
+#include "wallet/wallet.h"
+#include "amount.h"
+#include <memory>
+#include <string>
+#include <set>
+#include <vector>
+#include <boost/filesystem.hpp>
+
+// To run tests:
+// ./zcash-gtest --gtest_filter="founders_reward_test.*"
+
+//
+// Enable this test to generate and print 48 testnet 2-of-3 multisig addresses.
+// The output can be copied into chainparams.cpp.
+//
+#if 0
+TEST(founders_reward_test, create_testnet_2of3multisig) {
```

Comment:
My reading of this test is that the resulting testnet multisig addresses will be in a `wallet.dat` file in the filesystem's `tmp` folder? This should be documented in case the caller wants to be able to spend testnet FR for testing purposes.